lib/std/start.zig

```diff
@@ -215,6 +216,22 @@ fn posixCallMainAndExit() noreturn {
             std.os.linux.tls.initStaticTLS();
         }
 
+        if (!@hasDecl(root, "use_AT_RANDOM_auxval") or root.use_AT_RANDOM_auxval) {
+            // Initialize the per-thread CSPRNG since Linux gave us the handy-dandy
+            // AT_RANDOM. This depends on the TLS initialization above.
+            var i: usize = 0;
+            while (auxv[i].a_type != std.elf.AT_NULL) : (i += 1) {
```

Comment:
You can use `getauxval` here and avoid this loop, `elf_aux_maybe` is initialized above.
(No, this is not a hot path and no, I don't believe the open-coded loop is faster)